tokenserver/views.py

```diff
@@ -101,11 +101,14 @@ def valid_assertion(request):
     request.metrics['token.assertion.verify_success'] = 1
     request.validated['assertion'] = assertion
 
+    # pull out the FxA uid from the email address to log with the request
+    email = request.validated['assertion']['email']
+    request.metrics['id'] = email.split("@", 1)[0]
```

Comment:
That sounds fine to me. Let's go with `email` and then heka can do it's thing there.